packages/db/src/models/granule.ts

```diff
@@ -1,13 +1,21 @@
 import Knex from 'knex';
 
+import { RecordDoesNotExist, InvalidArgument } from '@cumulus/errors';
 import { tableNames } from '../tables';
-
 import { PostgresGranule, PostgresGranuleRecord, PostgresGranuleUniqueColumns } from '../types/granule';
 
 import { BasePgModel } from './base';
 import { GranulesExecutionsPgModel } from './granules-executions';
 import { translateDateToUTC } from '../lib/timestamp';
 
+interface RecordSelect {
+  cumulus_id: number
+}
+
+function isRecordSelect(param: RecordSelect | PostgresGranuleUniqueColumns): param is RecordSelect {
```

Comment:
I love this

Reply:
I'm a full-stack overflow developer. Actually not true, I found this in the typescript docs.